src/index.ts

```diff
@@ -0,0 +1,45 @@
+import fetch from 'node-fetch';
+
+import * as administrative from "../compute-typescript-fetch/administrative"
+import * as builder from "../compute-typescript-fetch/builder"
+import * as execution from "../compute-typescript-fetch/execution"
+
+if (!globalThis.fetch) {
+    // @ts-ignore
+	globalThis.fetch = fetch;
+}
+
+type config = {
+    baseUrl?: string;
```

Comment:
We probably need to set all three base URLs separately.

Reply:
Yes, however I think we should have some consts for the defaults used in the local and K8s deployments, since those are easily known to us. That way we can have a `suborbital.DefaultLocalConfig()` and `suborbital.DefaultK8sConfig()` that would work for 80% of people.